### DISCUSSION TITLE
How to Collapse All Components in Admin Menu by Default?

### DISCUSSION THREAD

**Discussion by giorgitchanturidze** [flyntwp/flynt#510] [Q&A] [answered]
I want all the components in the admin menu to be collapsed by default. Currently, when I navigate to the admin page, all components are expanded by default.

Can someone guide me on how to achieve this behavior, so I see all components closed and then choose which one to open?

**Comment by aaronmeder** [comment, marked as answer]
Hey @giorgitchanturidze 

I usually use this plugin: https://wordpress.org/plugins/acf-repeater-flexible-content-collapser/

It remembers the last state and reverts to that when loading an edit screen + adds a "collapse/expand all" button above the pageComponents.

**Reply by steffenbew** [reply]
We're using the same plugin.

Here's a list of ACF plugins that we like to use on our Flynt projects:
- [ACF Content Analysis for Yoast SEO](https://wordpress.org/plugins/acf-content-analysis-for-yoast-seo/)
- [ACF Hide Layout](https://wordpress.org/plugins/acf-hide-layout/)
- [ACF Repeater & Flexible Content Collapser](https://wordpress.org/plugins/acf-repeater-flexible-content-collapser/)
- [ACF Autosize](https://wordpress.org/plugins/acf-autosize/)

**Comment by giorgitchanturidze** [comment]
Thank you so much for the quick and detailed response!
